City-of-Helsinki/drupal-hdbt
# New TPR service list component with search

# [PLAENH-185](https://helsinkisolutionoffice.atlassian.net/browse/PLAENH-185)
New TPR service list component with search

## What was done
More details at
https://helsinkisolutionoffice.atlassian.net/browse/PLAENH-185

## How to install

* Update the module with this PR https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/128
* Update the theme with PR 
* Run `make drush-updb drush-cr`
* Run `PARTIAL_MIGRATE=1 drush migrate:import tpr_service --no-progress`


## How to test
* Add to a page that has paragraphs the new component "Service List Search"
* Configure the component with using service_id and/or specific services

* [ ] The page used in installation should have a listing that looks like this https://app.abstract.com/share/505f9e4a-9291-48ca-9bcc-4f38a7bc3e36?collectionId=4d66b7af-5224-4b71-8d19-a4df8af377c0&collectionLayerId=0d3595f3-dc9c-4c8e-8dc3-e7c5c09ac932&present=true&preview=false&sha=bf449ac81535612facaff26e7355b3b354509124
* [ ] Search should work against title, short desc and name synonyms
* [ ] Services teasers should use the imported translations

## Designers review



## Other PRs
* https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/128
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/407


[PLAENH-185]: https://helsinkisolutionoffice.atlassian.net/browse/PLAENH-185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
